src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -160,20 +142,24 @@ public double getAverageDistance() {
     return (getLeftDistance() + getRightDistance()) / 2;
   }
 
-  /**
-   * resets the encoder
-   */
-  public void resetEncoders() {
-    m_leftEncoder.reset();
-    m_rightEncoder.reset();
+  public void resetDriveTrain() {
+    resetDriveTrain(0.0, 0.0, 0.0);
```

Comment:
Please add a javadoc, like the one you have below for the corresponding `resetDriveTrain(double, double, double)` below.

Reply:
Resolved in 0bf8ea1

Reply:
"To the inputted position" doesn't make sense here. Should say something about x=0, y=0, angle=0

Reply:
resolved in 0929fdf